L1Trigger/TrackerDTC/python/ProducerES_cfi.py

```diff
@@ -7,7 +7,7 @@
     XMLLabel    = cms.string ("geomXMLFiles"                                    ), # label of ESProducer/ESSource
```

Comment:
Add comment to start of this PSet stating that this refers to files included by Configuration/Geometry/python/GeometryExtended*_cff.py

Reply:
done